### ISSUE TITLE
Write unit tests

### ISSUE THREAD

**Issue by alexlovescoding** [SAUGuild/FloorsProject#11]
Might be a good practice to write either gradle or maven unit tests to make sure new code doesn't break any existing code.

**Comment by jarylchong** [comment]
Yes yes yes yes yes. Unit testing is very important. I have done neither gradle nor maven tests, but we definitely should explore that.

**Comment by alexlovescoding** [comment]
I added the repo to Travis CI.  Now pull requests or pushes made to the master branch have to pass style checks and any tests in the test package.
